src/integrationTest/java/uk/gov/hmcts/reform/hmc/controllers/LinkHearingGroupControllerIT.java

```diff
@@ -552,7 +552,7 @@ void shouldReturn400_ForInvalidRequestId() throws Exception {
             mockMvc.perform(put(url + "?id=88782")
                                 .contentType(MediaType.APPLICATION_JSON_VALUE)
                                 .content(objectMapper.writeValueAsString(hearingLinkGroupRequest)))
-                .andExpect(status().is(400))
+                .andExpect(status().is(404))
```

Comment:
If the response code is changing - can we update the test name to reflect this?

Reply:
changed the file name